src/object-store/src/cache_policy.rs

```diff
@@ -19,12 +19,15 @@ use std::sync::Arc;
 use async_trait::async_trait;
 use bytes::Bytes;
 use lru::LruCache;
+use metrics::increment_counter;
 use opendal::ops::{OpDelete, OpList, OpRead, OpScan, OpWrite};
 use opendal::raw::oio::{Read, Reader, Write};
 use opendal::raw::{Accessor, Layer, LayeredAccessor, RpDelete, RpList, RpRead, RpScan, RpWrite};
 use opendal::{ErrorKind, Result};
 use tokio::sync::Mutex;
 
+use crate::metrics::*;
```

Comment:
Refer to https://pingcap.github.io/style-guide/rust/modules.html

> Avoid glob imports. Only use a glob import when you mean “import lots of items from some module”, not when you mean “I want lots of items from this module, but I don’t want to write them out”.
>
> Prefer to import modules, types, and macros; prefer not to import functions and constants, use qualified syntax

Use can import `metrics` mod if possible.
```suggestion
use crate::metrics;
```
However, if it conflicts with the `metrics` crate, it's ok to import the constants like this:
```rust
use crate::metrics::{OBJECT_STORE_LRU_CACHE_HIT, OBJECT_STORE_LRU_CACHE_MISS};
```

Reply:
@killme2008 Thanks for the reference. Are you ok with this?

Reply:
@nearsyh Yeah, it's all right. Sorry, I forgot that `crate::metrics` and `metrics` crate will conflict in this case.

Reply:
Done. Thanks for your help!